source/kit/components/ui/miscService.js

```diff
@@ -7,7 +7,10 @@
 
     function miscService() {
         return {
-            isPromise: isPromise
+            isPromise: isPromise,
+            ENTER_KEY: 13,
```

Comment:
I'd suggest to put these in a separate constant file (keys.constants.js or whatever) using angular constants. Deal?